### PR TITLE
Minor fixes in the French translation

### PR DESCRIPTION
**Pull request by Sylvhem**
- Change the translation of "Muted users" in the "column.mutes" from "Comptes muets" to "Comptes silencés".
- Minor typographic fixes.

### CODE REVIEW

File: app/assets/javascripts/components/locales/fr.jsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const fr = {
...
-  "compose_form.privacy_disclaimer": "Votre statut privé va être transmis aux personnes mentionnées sur {domains}. Avez-vous confiance en {domainsCount, plural, one {ce serveur} other {ces serveurs}} pour ne pas divulguer votre statut ? Les statuts privés ne fonctionnent que sur les instances de Mastodons. Si {domains} {domainsCount, plural, one {n'est pas une instance de Mastodon} other {ne sont pas des instances de Mastodon}}, il n'y aura aucune indication que votre statut est privé, et il pourrait être partagé ou rendu visible d'une autre manière à d'autres personnes imprévues",
+  "compose_form.privacy_disclaimer": "Votre statut privé va être transmis aux personnes mentionnées sur {domains}. Avez-vous confiance en {domainsCount, plural, one {ce serveur} other {ces serveurs}} pour ne pas divulguer votre statut ? Les statuts privés ne fonctionnent que sur les instances de Mastodons. Si {domains} {domainsCount, plural, one {n’est pas une instance de Mastodon} other {ne sont pas des instances de Mastodon}}, il n’y aura aucune indication que votre statut est privé, et il pourrait être partagé ou rendu visible d’une autre manière à d’autres personnes imprévues.",
```
Comment (wxcafe) [inline app/assets/javascripts/components/locales/fr.jsx]: would be nice to replace "instances de Mastodons" with "instances de Mastodon". Was wrong in the old version too, but if we're fixing stuff here let's do it thoroughly

File: app/assets/javascripts/components/locales/fr.jsx [hunk 2]
```diff
@@ -67,9 +67,9 @@ const fr = {
   "navigation_bar.edit_profile": "Modifier le profil",
   "navigation_bar.favourites": "Favoris",
   "navigation_bar.follow_requests": "Demandes de suivi",
-  "navigation_bar.info": "Plus d'informations",
+  "navigation_bar.info": "Plus d’informations",
   "navigation_bar.logout": "Déconnexion",
-  "navigation_bar.mutes": "Comptes muets",
+  "navigation_bar.mutes": "Comptes silencés",
```
Comment (wxcafe) [inline app/assets/javascripts/components/locales/fr.jsx]: I believe "silenciés" would be more natural - at least, I hear it more than "silencé" in casual speach

**Review by wxcafe — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/components/locales/fr.jsx (hunk 2 above)
Comment (trebmuh) [inline app/assets/javascripts/components/locales/fr.jsx]: After a quick search, "silenciés" doesn't look to exist. "silencés" does.

Comment (Sylvhem) [inline app/assets/javascripts/components/locales/fr.jsx]: Yes, "silencié" is a neologism while "silencé" is an actual word.

https://fr.wiktionary.org/wiki/silencer
http://www.cnrtl.fr/definition/silencer

File: app/assets/javascripts/components/locales/fr.jsx (hunk 1 above)
Comment (Sylvhem) [inline app/assets/javascripts/components/locales/fr.jsx]: You're right. I corrected that.

File: app/assets/javascripts/components/locales/fr.jsx (hunk 2 above)
Comment (wxcafe) [inline app/assets/javascripts/components/locales/fr.jsx]: meh, okay

**Review by wxcafe — APPROVED** [review]

**Pull request merged**